BatteryHealthMonitoring.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+#include <assert.h>
+#include "BatteryHealthMonitoring.h"
+#include "LimitedRangeParametersMonitoring.h"
+#include "ChargeRateMonitoring.h"
+#include "Config/ParameterConfig.h"
+
+#define TRUE 1
+#define FALSE 0
+
+
+float param_value[][3] = {
+  {25.0, 0.7, 70.0},  /* Good parameters */
+  {50.0, 0.0, 85.0},    /* Bad Parameters */
+  {(BMS_TEMPARATURE_HIGHER_WARNING_RANGE+0.5) ,
+   (BMS_CHARGERATE_HIGHER_WARNING_RANGE+0.01),
+   (BMS_STATEOFCHARGE_HIGHER_WARNING_RANGE+0.5)}  /*Warning Parameters */
+  };
+
+BatteryHealthControlData_t BatteryHealthControlData ={
```

Comment:
BatteryHealthControlData indicates that it's used to control the battery health?

Reply:
No. 
Actually wrong naming, Did not got any good name for the implementation. 
For your previous [comment ](https://github.com/clean-code-craft-tcq-1/functional-c-sembooooo/pull/1#discussion_r586564655)i have designed it like that so we just loop and call its methods (function pointers) instead of writing everything separately.
With the given three parameters as of now my code looks like an overkill.
I am not sure thats what you have asked or did i misunderstood it. 
So for such an implementation i did not got a good name and hence named it ControlData. But for the for loop coordinator logic.